.code-samples.meilisearch.yaml

```diff
@@ -132,6 +132,14 @@ update_settings_1: |-
   })
 reset_settings_1: |-
   client.index('movies').resetSettings()
+get_pagination_1: |-
+  client.index('movies').getPagination()
+update_pagination_1: |-
+  client.index('movies').updatePagination({
+    maxTotalHits: 100
+  })
+reset_pagination_1: |-
+  client.index('movies').resetPagination()
```

Comment:
This one is missing

as per the rust PR https://github.com/meilisearch/meilisearch-rust/pull/342/files: 

```
settings_guide_pagination_1: |-
  let pagination = PaginationSetting {max_total_hits:100};
  let task: TaskInfo = client
    .index("books")
    .set_pagination(pagination)
    .await
    .unwrap();
```

Reply:
Done